smarts/env/gymnasium/driving_smarts_competition_env.py

```diff
@@ -269,6 +283,34 @@ def _get_env_specs(scenario: str):
         raise Exception(f"Unknown scenario {scenario}.")
 
 
+def resolve_agent_action_space(agent_interface: AgentInterface):
+    if agent_interface.action == ActionSpaceType.RelativeTargetPose:
+        return gym.spaces.Box(
+            low=np.array([-28, -28, -np.pi]),
+            high=np.array([28, 28, np.pi]),
```

Comment:
Where did the number 28 come from?

Reply:
This was a constant based on the maximum safe deceleration... Oddly enough I am not certain why we are using it here since 280m/s is nonsensically fast.

Reply:
Oh, it was intended to be 2.8m/0.1s.

Reply:
Ok, maybe worth leaving a comment for clarity.

Reply:
I have declared the value as a commented constant.